app/uk/gov/hmrc/eeittadminfrontend/config/ApplicationLoader.scala

```diff
@@ -62,11 +61,11 @@ class ApplicationLoader extends play.api.ApplicationLoader {
 }
 
 class CustomHttpRequestHandler(
-    router: Router,
-    httpErrorHandler: HttpErrorHandler,
-    httpConfiguration: HttpConfiguration,
-    httpFilters: Seq[EssentialFilter]
-) extends DefaultHttpRequestHandler(router, httpErrorHandler, httpConfiguration, httpFilters: _*) {
+                                router: Router,
```

Comment:
Hmm, looks like intellij did formatting it's way.  Is there _scalariform_ plugin instlled in this project? If not just add it (and couple of it friends). Into plugins.sbt paste:

```scala
addSbtPlugin("org.scalariform"  % "sbt-scalariform" % "1.6.0")

addSbtPlugin("net.virtual-void" % "sbt-dependency-graph" % "0.8.2")

addSbtPlugin("com.timushev.sbt" % "sbt-updates" % "0.3.0") //sbt dependencyUpdates
```

Now every time when you compile project it will be reformatted in the same way as in other services. No to mention that it can save you from possible conflicts when rebasing/merging.